libopae/src/usrclk/user_clk_pgm_uclock.c

```diff
@@ -135,10 +136,17 @@ fpga_result __FIXME_MAKE_VISIBLE__ set_userclock(const char* sysfs_path,
 		return FPGA_NOT_SUPPORTED;
 	}
 
-	FPGA_DBG("User clock: %ld \n", refClock);
+	uint64_t refClk = 0;
```

Comment:
For consistency within the file, can we move the variable declaration to the top of the function?